metagpt/actions/design_api_an.py

```diff
@@ -45,13 +91,49 @@
     example=MMC2,
 )
 
+REFINED_PROGRAM_CALL_FLOW = ActionNode(
```

Comment:
This is a discussion, it seems that all increment or refined should refine the ActionNode, But most of the instruction are similar. So, Can we add a IncreActionNode/ RefineActionNode to follow a incre/refine **format** to extend the instruction, seems that will help the user to only handle the original one(PROGRAM_CALL_FLOW) but not with three(PROGRAM_CALL_FLOW INCRE_XX REFINED_XX).